test/fhirtypes/ElementDefinition.constrainType.test.ts

```diff
@@ -338,6 +339,46 @@ describe('ElementDefinition', () => {
       );
     });
 
+    it('should allow us to constrain a reference to a profile with a parent with a canonical including the version', () => {
```

Comment:
I know we came up w/ this test name together, but I found it confusing when I was reviewing.  I think this might be better:
```suggestion
    it('should allow us to constrain a reference to a profile whose parent is specified using a versioned canonical URL', () => {
```